src/getData.js

```diff
@@ -1,25 +1,28 @@
 const ulWrap = document.querySelector('.score-list');
 
-let arrOfScores = [];
-
-const previousData = JSON.parse(localStorage.getItem('storedScores'));
 const previousInput = JSON.parse(localStorage.getItem('storedInput'));
-
 if (previousInput) {
   [document.querySelector('.name-input').value,
     document.querySelector('.score-input').value] = previousInput;
 }
 
-if (!previousData) {
-  localStorage.setItem('storedScores', JSON.stringify(arrOfScores));
-} else {
-  arrOfScores = previousData;
-}
+export default function getData() {
+  const currentID = JSON.parse(localStorage.getItem('storedGameId'));
+  async function importData() {
+    const fetchScores = fetch(`https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${currentID}/scores`);
+    const update = await fetchScores;
+    const jsonObject = await update.json();
+    return jsonObject.result;
+  }
```

Comment:
- [ ] It would be better if you use arrow functions instead of function keyword. Please fix this minor issue. 

🛠️ 😄